bbb-common-web/src/main/java/org/bigbluebutton/api/util/ParamsUtil.java

```diff
@@ -19,11 +20,15 @@
   public static String stripControlChars(String text) {
     return text.replaceAll("\\p{Cc}", "");
   }
-  
+
   public static String stripHTMLTags(String value) {
     return value.replaceAll("\\<.*?>","");
   }
 
+  public static String escapeHTMLTags(String value) {
```

Comment:
Can we remove 

```
public static String stripHTMLTags(String value) {
    return value.replaceAll("\\<.*?>","");
  }
```

now?

Reply:
It's possible, I just keep it because if in future we wanted a function to strip tags it's already implemented,